src/main/java/sirius/biz/jobs/batch/file/DictionaryBasedArchiveImportJob.java

```diff
@@ -177,7 +178,9 @@ protected void importEntries() throws Exception {
                     handleMissingFile(importFile.filename, importFile.required);
                 }
 
-                handledFiles.add(importFile.filename);
+                if(!(Files.isConsideredHidden(importFile.filename) || Files.isConsideredMetadata(importFile.filename))) {
```

Comment:
**Required**: sure this check is in the right place? looks like ArchiveImportJob#extractAllFiles is the place to work on...